cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,261 @@
-describe('Web Tables page', () => {
-  it('', () => {
+import faker from 'faker'
 
+describe('Web Tables Test Suite', () => {
+  const generateWorker = () => {
+      return {
+      firstName: faker.name.firstName(),
+      lastName: faker.name.lastName(),
+      email: faker.internet.email(),
+      age: faker.random.number({ min: 20, max: 60 }),
+    };
+  };
+
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+
+  it('should test pagination', () => {
+
+    cy.get('select[aria-label="rows per page"]').select('5');
+
+    const workers = [];
+    for (let i = 0; i < 3; i++) {
+      const worker = generateWorker();
+      workers.push(worker);
+  
+      cy.get('#addNewRecordButton').click();
+  
+      cy.get('#firstName').type(worker.firstName);
+      cy.get('#lastName').type(worker.lastName);
+      cy.get('#userEmail').type(worker.email);
+      cy.get('#age').type(worker.age);
+      cy.get('#salary').type('12000');
+      cy.get('#department').type('Sales');
+  
+      cy.get('#submit').click();
+      cy.wait(2000);
+    }
+  
+      cy.get('.-btn').contains('Next').click();  
+      cy.wait(2000);
```

Comment:
Using `cy.wait()` without condition can lead to flaky tests. Wait for an actual condition to be met instead of waiting for an arbitrary period of time.